src/raftstore/store/worker/apply.rs

```diff
@@ -343,7 +348,7 @@ impl<'a> ApplyContextCore<'a> {
         }
     }
 
-    /// Finish applys for the delegate.
+    /// Finishes `Apply`s for the delegate.
```

Comment:
Maybe we need a place to explain what is "delegate" and what is "apply".

Reply:
I will add it later

Reply:
In this PR, or another?